frontend/src/components/widgets/options/ToolOptions.vue

```diff
@@ -126,7 +130,8 @@ export default defineComponent({
 					props: {},
 				},
 			],
-			Shape: [{ kind: "NumberInput", callback: this.setToolOptions, props: { value: 6, min: 3, isInteger: true, label: "Sides" } }],
+			Shape: [{ kind: "NumberInput", callback: this.setShapeOptions, props: { value: 6, min: 3, isInteger: true, label: "Sides" } }],
+			Line: [{ kind: "NumberInput", callback: this.setLineOptions, props: { value: 5, min: 1, isInteger: true, label: "Stroke width" } }],
```

Comment:
I believe we're using Title Case, so "Stroke width" should be "Stroke Width".

Reply:
Hah, I had that at first and wasn't sure. Fixing now.